documentation/assemblies/upgrading/assembly-upgrade-zookeeper.adoc

```diff
@@ -0,0 +1,14 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='assembly-upgrade-zookeeper-{context}']
+= Upgrading Kafka when using ZooKeeper
+
+[role="_abstract"]
+If you are using ZooKeeper for cluster management, a kafka upgrade requires an update to the Kafka version and the inter-broker protocol version.
```

Comment:
Maybe this would sound better? I think `using ZooKeeper for cluster management` might be a bit confusing for some.

```suggestion
If you are using ZooKeeper-based cluster, a Kafka upgrade requires an update to the Kafka version and the inter-broker protocol version.
```